default/python/AI/FreeOrionAI.py

```diff
@@ -3,12 +3,16 @@
 import pickle  # Python object serialization library
 import sys
 import random
+import logging
 
-from common.configure_logging import redirect_logging_to_freeorion_logger
-redirect_logging_to_freeorion_logger()
+from common.configure_logging import redirect_logging_to_freeorion_logger, convenience_function_references_for_logger
 
 import freeOrionAIInterface as fo  # interface used to interact with FreeOrion AI client  # pylint: disable=import-error
 
+log_level = logging.DEBUG if fo.getOptionsDBOptionBool("verbose-logging") else logging.INFO
```

Comment:
The logger code is common code and does not have access to the `fo.` server functions.  So at least the lookup has to remain in the AI code.

The universe generation and the AI won't necessarily have the same default log level.

Could we break up the FreeOrionAI into two modules: one that is the entry point for the C++ and a second that is the top-level code for the AI.  Currently, FreeOrionAI.py serves both purposes and we end up hiding initialization code in imports in order to make the AI code look nice.  If we separated the entry point from the AI controller, then the initialization code in the entry point would be expected, in a central location and easier to improve.